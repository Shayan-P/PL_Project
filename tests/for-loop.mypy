def sum(l=[], f=None):
    a = 0;
    for x in l:
        a = x + a;
        ;
    return a;
    ;

print(sum([1, 2, 3]))
print(sum([1, 10, 100]));
print(sum([]));
