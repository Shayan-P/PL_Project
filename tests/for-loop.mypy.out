(num-val 6)
(num-val 111)
(num-val 0)
'finished
